cypress/integration/homepage/home_page_current_series_spec.js

```diff
@@ -9,7 +9,10 @@ describe('Testing the Current Series on the Homepage:', function () {
       currentSeries = series;
     });
 
-    cy.ignorePropertyUndefinedTypeError();
+      cy.ignorePropertyUndefinedTypeError();
+      cy.on('uncaught:exception', (err, runnable) => {
+          return false
+      })
```

Comment:
@lynchl607 I'm seeing this particular block of code several times. Is this a fix for some odd behavior?